src/SAWScript/X86.hs

```diff
@@ -450,16 +451,17 @@ setSimulatorVerbosity verbosity sym = do
 doSim ::
   (?memOpts::Crucible.MemOptions, Crucible.HasLLVMAnn Sym) =>
   Options ->
+  FilePath ->
```

Comment:
Similarly, I think this `FilePath` argument is redundant.

Reply:
Quite a few of them probably are; I'll sort it out. Fixed in 52c9b64827039fecfe5fcb619845d8704b429710.